R/pull_NOMS_data.R

```diff
@@ -0,0 +1,50 @@
+library(googleAnalyticsR)
+library(googleAuthR)
+library(dplyr)
+library(arrow)
+library(tidyr)
+library(lubridate)
+library(plyr)
+library(urltools)
+
+get_nwisweb_google_analytics <- function(date_range = c('2020-01-01', '2020-02-01')) {
```

Comment:
Can you name this something like `get_nwis_site_page_views` so its a bit more descriptive?